ixmp/core.py

```diff
@@ -111,6 +111,12 @@ def set_log_level(self, level):
         logger().setLevel(level)
         self._backend.set_log_level(level)
 
+    def get_log_level(self):
+        """Get global logger level.
```

Comment:
"Get a log level associated with the storage :class:`.Backend`, if any."

Also add the "Returns" section as above.